new-cdc/eventuate-local-java-cdc-connector-db-log-common/src/main/java/io/eventuate/local/db/log/common/DbLogMetrics.java

```diff
@@ -62,12 +62,32 @@ public void eventReceived(long timestamp) {
     lag.set(System.currentTimeMillis() - timestamp);
   }
 
-  public void stop() {
-    if (meterRegistry == null) {
-      return;
-    }
+  public void onMessageProcessed() {
+    meterRegistry.counter("eventuate.messages.processed." + binlogClientId).increment();
```

Comment:
This pattern appears everywhere `prefix + binLogClientId` - perhaps there should be a method

Reply:
oh, yes, I will fix